referenceframe/frame_system.go

```diff
@@ -589,8 +589,17 @@ func createFramesFromPart(part *FrameSystemPart) (Frame, Frame, error) {
 		if err != nil {
 			return nil, nil, err
 		}
+		pose, err := staticOriginFrame.Transform([]Input{})
```

Comment:
This should all become moot when GetGeometries is fully implemented, but isn't a problem